### ISSUE TITLE
Authorization returns 401

### ISSUE THREAD

**Issue by bboriondev** [allegro/allegro-api#10217]
Hello,  I want to use the API, with the generated Client Id / Secret, but it returns 401 Unauthorized. I'm using the php example from the docs:
`function getAccessToken()
{
    $authorization = base64_encode(CLIENT_ID.':'.CLIENT_SECRET);
    $headers = array("Authorization: Basic {$authorization}","Content-Type: application/x-www-form-urlencoded");
    $content = "grant_type=client_credentials";
    $ch = getCurl($headers, $content);
    $tokenResult = curl_exec($ch);
    $resultCode = curl_getinfo($ch, CURLINFO_HTTP_CODE);
    curl_close($ch);
    if ($tokenResult === false || $resultCode !== 200) {
        exit ("Something went wrong $resultCode $tokenResult");
    }
    return json_decode($tokenResult)->access_token;
}
`

The client ID is 4fcbfdd799774097a78cd3408f3f449a4fcbfdd799774097a78cd3408f3f449a
How can I find out why the request fails?
Thank you in advance!

**Comment by MaciejFrackowiak** [comment]
Please, check the provided data again - especially encoded data (ClientID and Client Secret with ":" between them). Our example should work without any issues. Also, check [our tutorial](https://developer.allegro.pl/tutorials/authentication-and-authorization-m09BlVyo7iY#clientcredentials-flow) once more.

**Comment by bboriondev** [comment]
I'm using the method copied from the tutorial, which encodes the two constants - I tried it with another cliend it / secret key pair, same result.

**Comment by MaciejFrackowiak** [comment]
Please, send us the sample data which you provided - you can do it via the [Allegro contact form](https://allegro.pl/pomoc/kontakt) (add GitHub thread number).

**Comment by bboriondev** [comment]
Hello, I sent it, but they didn't reply for more than 2 weeks... Do you have any other contact method?

**Comment by bboriondev** [comment]
Hello, did you get the message?
I'm not sure how the contact form works, I'm using google translate - after I write the message, select the API, it redirects to a summary page, but I don't know weather the message is sent, or it requires any further action from my behalf...
![image](https://github.com/user-attachments/assets/d87fb682-bb85-4b34-aeed-97d221f0d560)

**Comment by MaciejFrackowiak** [comment]
No, I haven't. Please, choose a different topic for your message and paste your data. Add GitHub thread number #10217.

**Comment by bboriondev** [comment]
So first I sent a question, they said I should write on github.
You said to send the details via the form, but since the API topic wouldn't send the message, I sent it as some other business related topic.
Now I got a reply that I SHOULD ASK THE DEVELOPMENT QUESTIONS ON GITHUB.
You guys are a marvel, seriously.

So here I am, writing on Github. What now, should I write on the contact form? :)

**Comment by MaciejFrackowiak** [comment]
I'm sorry for such a situation. However, you shouldn't paste private data (e.g. tokens) here because it is sensitive data, so I asked you to send it to us via the Allegro contact form. So once again please contact them and ask them to redirect your data directly to us - add the GitHub thread number https://github.com/allegro/allegro-api/issues/10217. Of course, give us a new example of your request to our server.

**Comment by MaciejFrackowiak** [comment]
I just wanted to let you know that I have received your email.
First, you have to change the URL address - "https://allegro.pl/auth/oauth/token" instead of "https://allegro.hu/auth/oauth/token". Also, the Client ID was incorrect - you provided the value twice.
And with that changes, you should authorize correctly with client_credentials auth - in [our tutorial](https://developer.allegro.pl/tutorials/authentication-and-authorization-m09BlVyo7iY#clientcredentials-flow), we described the process step by step with sample codes.
